### DISCUSSION TITLE
How can I use a persistent volume for a shared file system snapshot repository?

### DISCUSSION THREAD

**Discussion by nilushancosta** [Opster/opensearch-k8s-operator#858] [Q&A] [answered]
I want to add a shared file system snapshot repository to my OpenSearch cluster deployed by the operator. I was hoping to mount a Persistent Volume with ReadWriteMany access policy to the cluster (E.g. NFS, Azure File) by using [additionalVolumeMounts](https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/docs/userguide/main.md#additional-volumes). However I couldn't find a volume type suitable for it. 
Is there a different way to add such a volume for snapshot repositories?

**Comment by swoehrl-mw** [comment, marked as answer]
Hi @nilushancosta . Currently this is not possible, unless you can specify your volume as a CSI volume. 
There is an open PR #852. Is that what you are looking for?

**Reply by nilushancosta** [reply]
Thanks for the response @swoehrl-mw . Yes that PR addresses what I am looking for. I will wait for it
